code/modules/shuttle/emergency.dm

```diff
@@ -343,7 +343,10 @@
 	icon_state = "dorm_available"
 	density = 0
 
-/obj/machinery/computer/shuttle/pod/update_icon()
+/obj/machinery/computer/shuttle/pod/update_icon_state()
+	return
+
+/obj/machinery/computer/shuttle/pod/update_overlays()
 	return
```

Comment:
```suggestion
	return ..()
```

Reply:
To follow with the comment on the original proc, this should return a list. I'm avoiding calling the parent on purpose.